Add Cemara Mania Cup #1 (2020) wiki article (#4694)
ppy/osu-wiki: wiki/Tournaments/CMC/2020/en.md

---
tags:
  - CMC
  - CMC 2020
---

# Cemara Mania Cup \#1 (2020)

![CMC 2020 logo](img/logo.jpg)

> The **Cemara Mania Cup \#1 (2020)** (***CMC 2020***) was a double-elimination 2v2 osu!mania 4K tournament hosted by ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) and co-hosted by ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653). The tournament was open to all osu!mania players from Indonesia with a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower. It was the 1st iteration of the Cemara Mania Cup.

Reviewer: ```suggestion
The **Cemara Mania Cup \#1 (2020)** (***CMC 2020***) was a double-elimination 2v2 osu!mania 4K tournament hosted by ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) and co-hosted by ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653). The tournament was open to all osu!mania players from Indonesia with a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower. It was the first iteration of the Cemara Mania Cup.
```
not sure if this was an intended spelling for tourney articles; ignore if so

Author: i think both are applicable (i've seen that some articles are using cardinal numbers and some others are using ordinal numbers instead) but yeah why not \:P

![image](https://user-images.githubusercontent.com/36564236/103252011-b3399f80-49ad-11eb-9a38-8af7c4d94d12.png)

![image](https://user-images.githubusercontent.com/36564236/103252035-bfbdf800-49ad-11eb-8dc4-055a63721919.png)

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2020-10-06/2020-11-01 |
| Screening phase | 2020-11-02/2020-11-08 |
| Qualifiers | 2020-11-09/2020-11-15 |
| Group Stage | 2020-11-16/2020-11-22 |
| Quarterfinals | 2020-11-23/2020-11-29 |
| Semifinals | 2020-11-30/2020-12-06 |
| Finals week 1 | 2020-12-07/2020-12-13 |
| Finals week 2 | 2020-12-14/2020-12-20 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 3 months of osu!supporter tag for each team member + 100% of the raised prize pool |

Reviewer 2: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 3 months of osu!supporter tag for each team member, 100% of the raised prize pool |
```

| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag for each team member |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag for each team member |

> | *4th place* | Steam Wallet voucher worth approx. IDR 50.000,- for each team member |

Reviewer 2: ```suggestion
| *4th place* | Steam Wallet voucher worth approx. IDR 50.000 for each team member |
```

Author: i think i'll leave this as is since on [the ICC wiki article](http://osu.ppy.sh/wiki/en/Tournaments/ICC/2020) (which was also handled by me back then) the prize money has been specified like that

![image](https://user-images.githubusercontent.com/36564236/103252426-74a4e480-49af-11eb-96ed-5ba629a36531.png)

| Referee | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [Fyl](https://osu.ppy.sh/users/10069307), ![][flag_ID] [xChose](https://osu.ppy.sh/users/8738085), ![][flag_ID] [Kurami](https://osu.ppy.sh/users/10184990), ![][flag_ID] [-Nana-](https://osu.ppy.sh/users/8738085), ![][flag_ID] [LyfLin](https://osu.ppy.sh/users/5198950) |
| Graphic designer | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [-Nana-](https://osu.ppy.sh/users/8738085) |
| Spreadsheet manager | ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949), ![][flag_ID] [Sies](https://osu.ppy.sh/users/6491991), ![][flag_ID] [Ririya-](https://osu.ppy.sh/users/15590443) |
| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |

## Links

> - **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977)**
> - [Forum thread](https://osu.ppy.sh/community/forums/topics/1157755)
> - [Discord server](https://discord.gg/5a2dy53)
> - [Livestream channel](https://www.twitch.tv/osuIndonesia)
> - [Challonge bracket](https://challonge.com/CMC_1)

Reviewer: ```suggestion
- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977 "Google Spreadsheet")**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1157755)
- [Discord server](https://discord.gg/5a2dy53 "Discord")
- [Livestream channel](https://www.twitch.tv/osuIndonesia "Twitch")
- [Challonge bracket](https://challonge.com/CMC_1 "Challonge")
```

Author: not sure if this is needed since i've never seen any other tournament article implementing this actually

## Podium

> Cemara Mania Cup \#1 (2020) has officially been concluded with the following podium:

Reviewer 2: ```suggestion
This competition has come to an end and resulted in the following podium:
```

  - [Dormir - Une mage blanche (cjunh4972) \[Priere de guerison\]](https://osu.ppy.sh/beatmapsets/856084#mania/1788654)
  - [An - artcore JINJA (Abraxos) \[Quiescence\_Lv.21\]](https://osu.ppy.sh/beatmapsets/409503#mania/888942)    
  - [The Black Dahlia Murder - Deathmask Divine (Valedict) \[Necromance\]](https://osu.ppy.sh/beatmapsets/563020#mania/1190604)
  - [AAAA - Hoshizora no Kanransha (Len) \[LeiN-'s 4K Insane\]](https://osu.ppy.sh/beatmapsets/877047#mania/1837281)   
  - [Frequent - Lucid (feat. Jan Amit) (Elekton) \[lull\]](https://osu.ppy.sh/beatmapsets/901710#mania/1882727)
  - [Shawn Wasabi - Marble Soda (-mint-) \[Ramune\]](https://osu.ppy.sh/beatmapsets/946991#mania/1977877)
  - [Veil of Maya - Mikasa (Shoegazer) \[Salvation\]](https://osu.ppy.sh/beatmapsets/828101#mania/1735096)
  - [seafoxy - Hisame (FelixSpade) \[New Season\]](https://osu.ppy.sh/beatmapsets/1091518#mania/2281898)
- Tiebreaker
  - **[Magnetude - Broken (SaltyMermaid) \[Shattered\]](https://osu.ppy.sh/beatmapsets/842099#mania/1762138)** 

### Group Stage

**[Download the mappack here! (29 MB)](https://mega.nz/file/WJYlVAyB#VaveHrRewqSkukDhdYajrhvWeS7CY-iZ38j4Pc7s0ok)**

- FreeMod

>   - [Getter Jaani - Rockefeller Street (\[GS\]Logan) \[ProSVperity\]](https://mega.nz/file/jdAwXRIB#tFUChm5qXEYcnhds5ShZu_y4qd06aYjjbn7MOHOuJDM)

Reviewer: ```suggestion
  - [Getter Jaani - Rockefeller Street (\[GS\]Logan) \[ProSVperity\]](https://mega.nz/file/jdAwXRIB#tFUChm5qXEYcnhds5ShZu_y4qd06aYjjbn7MOHOuJDM "MEGA")
```

Author: same as http://github.com/ppy/osu-wiki/pull/4694#discussion_r549535270

  - [Ben Briggs - Fearless Living (RuleBlazing) \[Coward\]](https://osu.ppy.sh/beatmapsets/1283930#mania/2666220)
  - [Thaehan - Final Boss (Theresa May) \[Stage 4 : Credits\]](https://osu.ppy.sh/beatmapsets/980090#mania/2051097)   

### Qualifiers

**[Download the mappack here! (22 MB)](https://mega.nz/file/3ZJBzQ6Y#_zFdkYDEkMnSckC9bx_Sl_0358CHZU_ZWOEPtfz2yvI)**

- FreeMod
  - [Shawn Wasabi + YDG - Burnt Rice (HowToPlayLN) \[Overcooked\]](https://osu.ppy.sh/beatmapsets/995871#mania/2083163)
  - [Yuzuki - Dear You (Vocal Version) (RemFangirl) \[Distant\]](https://osu.ppy.sh/beatmapsets/763326#mania/1605032)
  - [USAO - Knight Rider (Spy) \[Mashiro's NORMAL\]](https://osu.ppy.sh/beatmapsets/1007938#mania/2138728)
  - [xi - Titania (Murumoo) \[Lirai's EXPERT\]](https://osu.ppy.sh/beatmapsets/1174746#mania/2487983)
  - [DETRO - volcanic (\[Crz\]Crysarlene) \[wrong one\]](https://osu.ppy.sh/beatmapsets/984123#mania/2059102)  

## Match results

### Finals week 2

Saturday, 19 December 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | UwU Team | 2 | **7** | **Leviathan** | [#1](https://osu.ppy.sh/community/matches/71838770) |

> Sunday, 13 December 2020, Grand Final:

Reviewer 2: ```suggestion
Sunday, 13 December 2020 (Grand Final):
```

| Upper | UwU Team | 0 | **7** | **Neppu-Neppu** | [#1](https://osu.ppy.sh/community/matches/71412828) |

Sunday, 13 December 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | **Leviathan** | **7** | 3 | Team 11 | [#1](https://osu.ppy.sh/community/matches/71475447) |

### Semifinals

Friday, 4 December 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | Team 3 | 2 | **6** | Leviathan | [#1](https://osu.ppy.sh/community/matches/70941169) |

Saturday, 5 December 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | L Hunter | 2 | **6** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70998868) |
| Upper | SCP Indonesia | 2 | **6** | **UwU Team** | [#1](https://osu.ppy.sh/community/matches/71003960) |
| Upper | **Neppu-Neppu** | **6** | 3 | GWS | [#1](https://osu.ppy.sh/community/matches/71008403) |

Sunday, 6 December 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | GWS | 2 | **6** | Team 11 | [#1](https://osu.ppy.sh/community/matches/71071493) |

> | Lower | SCP Indonesia | 0 | **6** | **Leviathan** | *win by default* |

Reviewer: should "w" be capitalised here? and should "win" be past tense?

Author: this is actually how it is supposed to be written. even in other tournament articles (including the World Cup pages) the "*win by default*" here has been always left as is to properly indicate the match output

![image](https://user-images.githubusercontent.com/36564236/103252171-378c2280-49ae-11eb-90de-95056a1c65fb.png)

| Lower | Team 16 | 0 | **5** | **Soundtoloyo** | [#1](https://osu.ppy.sh/community/matches/70600889) |
| Upper | Leviathan | 2 | **5** | **Neppu-Neppu** | [#1](https://osu.ppy.sh/community/matches/70605490) |
| Upper | Team 1 | 0 | **5** | **SCP Indonesia** | [#1](https://osu.ppy.sh/community/matches/70598560) |
| Lower | **Pecel Lele Ujang** | **5** | 1 | Anti LN Club | [#1](https://osu.ppy.sh/community/matches/70601346) |
| Upper | L Hunter | 4 | **5** | **UwU Team** | [#1](https://osu.ppy.sh/community/matches/70611342) |
| Upper | Team 3 | 2 | **5** | **GWS** | [#1](https://osu.ppy.sh/community/matches/70601710) |

Sunday, 29 November 2020:

| Bracket | Team 1 |  |  | Team 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Lower | Team 1 | 0 | **5** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70660894) |
| Lower | **L Hunter** | **5** | 0 | Pecel Lele Ujang | [#1](https://osu.ppy.sh/community/matches/70666560) |
| Lower | **Team 3** | **5** | 2 | Soundtoloyo | [#1](https://osu.ppy.sh/community/matches/70600889) |
| Lower | **Leviathan** | **5** | 0 | Sumber Rejeki | [#1](https://osu.ppy.sh/community/matches/70663842) |

## Ruleset

### General rules

> 1. Map scoring is based on **Score V2.**

Reviewer: ```suggestion
1. Map scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
```

2. The mapsets for each round will be announced by the Tournament Management in advance before the actual matches take place.

> 3. Match schedules will be predetermined by the Tournament Management. Had there been any team(s) who were unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the #reschedule channel in the tournament's Discord server.

Reviewer: this should prob keep consistent tense; it flip flops between present and past. this goes for the rest of the list as well

4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.

> 5. If both members of a team does not show up within **10 minutes** of the start time, their opponent wins by default.

Reviewer: ```suggestion
5. If both members of a team do not show up within **10 minutes** of the start time, their opponent wins by default.
```

   - If a team can only institute one team member (out of the required two) for the match, the referee can either continue the match in a 2v1 setting (at the opposing team's discretion) or prerogatively award their opponent a Win By Default.
6. If no staff or referee is available, the match will be postponed.
7. If a player disconnects, it will be treated as if they failed the map.

>    - Disconnects that occur within a few seconds after map is started by the referee can be rematched.
> 8. If a player disconnects between the beatmaps, the match can be delayed up to 15 minutes max.

Reviewer: ```suggestion
   - Disconnects that occur within a few seconds after the map is started by the referee can be rematched.
8. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes max.
```

   - In case the disconnected player failed to report back after the maximum allowance of 15 minutes had passed, the referee can either continue the match in a 2v1 setting (at the opposing team's discretion) or prerogatively award their opponent a Win By Default.
9. Lag is not a valid reason to nullify a map.

> 10. If there are any unforseen problems during the match occurence, the Tournament Management will make a decision based on referee's report.

Reviewer: ```suggestion
10. If there are any unforseen problems during a match occurence, the Tournament Management will make a decision based on the referee's report.
```

> 11. It is expected to be polite and respectful to each other. Penalties will be given if players violate.
>     - If a player is found to be engaging in an act of offensive provocation, the said player responsible for the provocation may be either disqualified right away from CMC \#1 (2020) and/or blacklisted from the future installment(s) of CMC.

Reviewer: ```suggestion
11. It is expected that all players be polite and respectful to each other at all times. Penalties will be given if players violate.
    - If a player is found to be engaging in an act of offensive provocation, said player responsible for the provocation may be either disqualified right away from CMC \#1 (2020) and/or blacklisted from the future installment(s) of CMC.
```

>     - Usage of any illegal programs is strictly prohibited and will be straight up reported to the osu!staff at will.

Reviewer: ```suggestion
    - Usage of any illegal programs is strictly prohibited and will be immediately reported to the osu!staff at will.
```
"straight up" sounds a little informal imo

>     - If a player is found to be smurfing another player's account, the corresponding players will be blacklisted in the next installment(s) of CMC regardless of their role in the smurfing accident.

Reviewer: ```suggestion
    - If a player is found to be smurfing another player's account, the corresponding players will be blacklisted in the next installment(s) of CMC regardless of their role in the smurfing incident.
```
not sure if "accident" was intentional or not; ignore if so

### Tournament registration

1. All interested players  are required to register into the tournament individually.
   - In order to be eligible to play in the tournament, a player must have a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower and the ![][flag_ID] Indonesian flag displayed on their profile.
2. To ensure valid and serious registrations, every registered player will be checked by the Tournament Management and screened by osu!'s Tournament Staff.
3. The list of players who have passed the screening and are able to compete in the tournament will be published after the Registration Phase has ended.
4. Testplayers, referees, and map selectors may not participate as players in this tournament.

### Qualifier rules

1. Each player will have to sign up to one of the Qualifier lobbies that has been scheduled and prepared by the Tournament Management in advance.

> 2. In the lobby, all players will have to consecutively play all of the five Qualifier maps in the order that it is being written on the [main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977).

Reviewer: ```suggestion
2. In the lobby, all players will have to consecutively play all of the five Qualifier maps in the order that it is being written on the [main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977 "Google Spreadsheet").
```

   - All maps are due to be played twice in each lobby; on each map, only the best score from the two attempts that will be taken into consideration.
3. Players **are not allowed** to ban any maps in the Qualifiers.

> 4. Players **are not allowed** to join in (or register for) more than one Qualifier lobbies.

Reviewer: ```suggestion
4. Players **are not allowed** to join in (or register for) more than one Qualifier lobby.
```

5. 32 players with the **highest amount of accumulated Qualifier score in total** will advance to the Group Stage.
   - Players who ranked #1 - #8 based on their accumulated Qualifier score will be designated as Seed A players.
   - Players who ranked #9 - #16 based on their accumulated Qualifier score will be designated as Seed B players.
   - Players who ranked #17 - #24 based on their accumulated Qualifier score will be designated as Seed C players.
   - Players who ranked #18 - #32 based on their accumulated Qualifier score will be designated as Seed D players.
   - Players who ranked #33 or lower based on their accumulated Qualifier score will be knocked out from the tournament.
6. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.

### Group Stage rules

> 1. In the Group Stage, the 32 previously qualified players will be randomly spread out into 8 groups consisting of 4 different players each based on their seed.

Reviewer: ```suggestion
1. In the Group Stage, the 32 previously qualified players will be randomly spread out into eight groups consisting of four different players each based on their seed.
```

2. All the players from each group will face each other in a single multiplayer lobby under the "Battle Royale" matchup system as follows:

>    - All players will have to consecutively play all of the six Group Stage maps in the order that it is being written on the [main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977).

Reviewer: ```suggestion
   - All players will have to consecutively play all of the six Group Stage maps in the order that it is being written on the [main spreadsheet](https://docs.google.com/spreadsheets/d/1VttS6M8GdWINmMBuHws6AcND0yh3mNGzUiVQwleLP4Q/edit#gid=1467418977 "Google Spreadsheet").
```

>    - After the conclusion of each map, the two players that performs the best will receive +2 points while the other two players will receive +1 point.

Reviewer: ```suggestion
   - After the conclusion of each map, the two players that perform the best will receive +2 points while the other two players will receive +1 point.
```

3. Players **are not allowed** to ban any maps in the Group Stage.
4. Players will subsequently be ranked in their respective groups based from the amount of points they accumulate in the room. 
   - In case of a points tie, all affected players will be ordered by their Group Stage seed.

### Knock-out stage rules

1. Based on each player's Group Stage finishing position, players will then be teamed up into pairs of two where the group winners are set to be matched with the group runners-up and the 3rd place finishers are set to be matched with the 4th place finishers at random.
2. Teams will compete against each other using the Double Elimination system.

> 3. The Double Elimination System works as following:

Reviewer: ```suggestion
3. The Double Elimination System works as follows:
```

   - Teams who lose in the Upper Bracket can still play again on the Lower Bracket.
   - Teams who lose in the Lower Bracket will be eliminated from the tournament.
   - In the Grand Finals, the winner of the the Upper Bracket only needs to win a single match against their opponent in order to claim the championship title. The winner of the Lower Bracket, however, needs to win two matches and enforce a Bracket Reset against their opponent in order to claim the championship title.

> 4. Teams who can compete in the next round are determined by:

Reviewer: ```suggestion
4. Teams who can compete in the next round are determined by the following criteria:
```

   - In the Quarterfinals, each team needs to win 5 points in order to win a match. (Best-of-9)
   - In the Semifinals, each team needs to win 6 points in order to win a match. (Best-of-11)
   - In both of the Finals weeks, each team needs to win 7 points in order to win a match. (Best-of-13)   
   - Team(s) who Win By Default.
   - Whether there are team(s) who got disqualified during the match.

### Match regulations

> 1. A representative of each team must use the `!roll` command once in #multiplayer in order to determine the banning and picking order.

Reviewer: #mulitplayer in the in-game chat or the discord server? it would be useful to clarify

   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier and Group Stage lobbies.
2. Each team has to ban **two beatmaps** from the mappool. These beatmaps are not allowed to be picked by any team during the entire match.
   - Banning does not apply in the Qualifier and Group Stage lobbies.
3. Each team is free to select one warm-up beatmap. Using beatmaps with questionable content is prohibited.
   - Warm-ups do not apply in the Qualifier and Group Stage lobbies.
4. All maps (including the Tiebreaker) will be played with the FreeMod option enabled. Each player *may* opt to play a map with either Hidden, FadeIn, Mirror, Flashlight, or any possible combinations of the four mods.
   - Playing a map without any mods activated whatsoever is allowed, however playing a map with a mod that is not mentioned above (i.e. Random) is **not** allowed.

> 5. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match has concluded by the responsible referees.

Reviewer: ```suggestion
5. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match is concluded by the responsible referees.
```
